stochastic_physics.F90

```diff
@@ -23,7 +23,7 @@ subroutine init_stochastic_physics(levs, blksz, dtp, sppt_amp, input_nml_file_in
     xlon,xlat, &
     do_sppt_in, do_shum_in, do_skeb_in, lndp_type_in, n_var_lndp_in, use_zmtnblck_out, skeb_npass_out,    &
     lndp_var_list_out, lndp_prt_list_out,                         &
-    n_var_spp_in, spp_var_list_out, spp_prt_list_out, do_spp_in,     &
+    n_var_spp_in, spp_var_list_out, spp_stddev_cutoff_out, spp_prt_list_out, do_spp_in,     &
```

Comment:
Thanks.  I'll make this modification.  Does it matter in terms of the variables themselves?  My print statements of these fields in the Thompson MP routine looked OK.

Reply:
The order in the call stochastic_physics_wrapper.F90 has to match the order in the argument list in stochastic_physics.F90, otherwise you'll send the wrong value to the wrong variable.  That's because the call uses positional arguments. If the call used named arguments, the order would not matter.

Reply:
The way it is coded, the variables would be swapped.

Reply:
Just fixed the order.